### DISCUSSION TITLE
Kibana doesnt show older logs

### DISCUSSION THREAD

**Discussion by Foxsden13** [telekom-security/tpotce#1388] [Q&A]
Hey guys,
I really would appreciate your help on that one.
I had the T-Pot running for three weeks and created a screenshot of the server instance every seven days.
This was in July (5th-26th).
Today I was trying to analyze the logs but unfortunately, Kibana wont show them anymore.
The log files must still be there, because the /opt/data folder is almost the size of 80GB. Same as when I created the screenshot.
Maybe it has something to do with the feature 'logrotate'?
Maybe someone has another idea?

Best regards!

**Comment by RytterMohn** [comment]
Hello, have you solved the problem?
